js/src/containers/PersistentDismissableAlert.js

```diff
@@ -0,0 +1,30 @@
+import { Alert } from "@yoast/components";
+import { compose } from "@wordpress/compose";
+import { withDispatch, withSelect } from "@wordpress/data";
+
+/**
+ * Composes the PersistentDismissableAlert container.
+ *
+ * @returns {Component} The composed Alert component.
+ */
+export default compose( [
+	withSelect( ( select, ownProps ) => {
+		const {
+			isAlertDismissed,
+		} = select( "yoast-seo/editor" );
+
+		return {
+			isAlertDismissed: isAlertDismissed( ownProps.alertKey ),
+		};
+	} ),
+
+	withDispatch( ( dispatch, ownProps ) => {
+		const {
+			dismissAlert,
+		} = dispatch( "yoast-seo/editor" );
+
+		return {
+			onDismissed: () => dismissAlert( ownProps.alertKey ),
+		};
+	} ),
+] )( Alert );
```

Comment:
It seems very strange to me to add a prop to Alert that if it is false, nothing is rendered.
But then again, it does have the dismiss action.

If you would want to do it differently, you could make a small intermediate component here that does not render `Alert`, something like:

```js
function PersistantDismissableAlert( { isAlertDismissed, ...restProps } ) {
	return isAlertDismissed === false && <Alert { ...restProps } />;
}
```

Reply:
Both approaches have their pros and cons. An extra in between component doesn't seem a lot better to me at the moment, so keeping this approach for now.